src/Api/Controllers/OrganizationExportController.cs

```diff
@@ -52,13 +52,13 @@ private ListResponseModel<CollectionResponseModel> GetOrganizationCollectionsRes
             return new ListResponseModel<CollectionResponseModel>(collections);
         }
 
-        private async Task<ListResponseModel<CipherMiniDetailsResponseModel>> GetOrganizationCiphersResponse(IEnumerable<CipherOrganizationDetails> orgCiphers,
+        private Task<ListResponseModel<CipherMiniDetailsResponseModel>> GetOrganizationCiphersResponse(IEnumerable<CipherOrganizationDetails> orgCiphers,
```

Comment:
I think the proper solution here is to remove Task, and just return the response directly? It's not an async operation.

Reply:
@Hinton I have a PR with this fix here: https://github.com/bitwarden/server/pull/2215